### PR TITLE
Routine maintenance of the docs

### LINKED ISSUE #582: Add physical costing notebook(s) to the docs
following https://github.com/quantumlib/Qualtran/pull/568

need to add them to the table of contents somewhere

### PR DESCRIPTION
**Pull request by mpharrigan**
- sync autogenerated notebooks
 - include new notebooks in the sphinx tocs
 - add some more notes to the developer docs for how to do doc stuff
 - fix some mathjax incompatibilities

Fixes #582 

these are things that can be more automated #330

### CODE REVIEW

File: dev_tools/autogenerate-bloqs-notebooks.py [hunk 1]
```diff
@@ -129,25 +129,16 @@
...
-    NotebookSpec(
-        title='Sparse Hamiltonian',
-        module=qualtran.bloqs.chemistry.sparse,
-        gate_specs=[
-            BloqNbSpec(qualtran.bloqs.chemistry.sparse.prepare_test._make_sparse_prepare),
-            BloqNbSpec(qualtran.bloqs.chemistry.sparse.select_test._make_sparse_select),
-        ],
-        directory=f'{SOURCE_DIR}/bloqs/chemistry/sparse',
-    ),
+    # NotebookSpec(
```
Comment (NoureldinYosri) [inline dev_tools/autogenerate-bloqs-notebooks.py:132]: is this supposed to be deleted?

**Review by NoureldinYosri — APPROVED** [review]

File: dev_tools/autogenerate-bloqs-notebooks.py (hunk 1 above)
Comment (mpharrigan) [reply dev_tools/autogenerate-bloqs-notebooks.py:132]: The deleted stuff is supposed to be deleted, as it has been migrated to the `-v2` script.

@fdmalone should chime in: but it looks like the commented out part has been _partially_ migrated to the `-v2` script. The commented-out code does not run; it contains references to deleted example-making functions

**Comment by mpharrigan** [comment]
changing to draft so I can incorperate #578 (and #583)

File: dev_tools/autogenerate-bloqs-notebooks.py (hunk 1 above)
Comment (fdmalone) [reply dev_tools/autogenerate-bloqs-notebooks.py:132]: you should probably just delete the commented out code too?

Comment (fdmalone) [reply dev_tools/autogenerate-bloqs-notebooks.py:132]: oh it hasn't been migrated, let me fix that.

**Comment by mpharrigan** [comment]
ok this has been rebased following the chemistry contributions and is simpler as a result

**Review by fdmalone — APPROVED** [review]

**Pull request merged**